src/app/story/directives/basic.component.ts

```diff
@@ -48,6 +48,17 @@ import { StoryModel, TabService } from '../../shared';
       tiptext="More is better!" >
         <div class="fancy-hint">A comma-separated list of tags relevant to the content of your piece.</div>
       </publish-fancy-field>
+      
+      <publish-fancy-field label="Release Date">
```

Comment:
I added changed support, but not invalid. Typing garbage into the input box does not change the model, but it should show invalid.